spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -1,87 +1,99 @@
-          uuid: xx-1-0-0-op-s-bm-002
+          uuid: ALA-1-0-0-op-s-bm-002
         - service-name: /v1/redirect-oam-request-information
-          uuid: xx-1-0-0-op-s-bm-003
+          uuid: ALA-1-0-0-op-s-bm-003
         - service-name: /v1/end-subscription
-          uuid: xx-1-0-0-op-s-bm-004
+          uuid: ALA-1-0-0-op-s-bm-004
         - service-name: /v1/inquire-oam-request-approvals
-          uuid: xx-1-0-0-op-s-bm-005
+          uuid: ALA-1-0-0-op-s-bm-005
         - service-name: /v1/update-client
-          uuid: xx-1-0-0-op-s-bm-007
+          uuid: ALA-1-0-0-op-s-bm-007
         - service-name: /v1/list-ltps-and-fcs
-          uuid: xx-1-0-0-op-s-bm-008
+          uuid: ALA-1-0-0-op-s-bm-008
         - service-name: /v1/redirect-topology-change-information
-          uuid: xx-1-0-0-op-s-bm-009
+          uuid: ALA-1-0-0-op-s-bm-009
         - service-name: /v1/update-operation-key
-          uuid: xx-1-0-0-op-s-bm-010
+          uuid: ALA-1-0-0-op-s-bm-010
         - service-name: /v1/update-operation-client
-          uuid: xx-1-0-0-op-s-bm-011
+          uuid: ALA-1-0-0-op-s-bm-011
       individual:
         - service-name: /v1/bequeath-your-data-and-die
-          uuid: xx-1-0-0-op-s-im-000
+          uuid: ALA-1-0-0-op-s-im-000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: ALA-1-0-0-op-s-bs-000
         - service-name: /v1/inform-about-application
-          uuid: xx-1-0-0-op-s-bs-001
+          uuid: ALA-1-0-0-op-s-bs-001
         - service-name: /v1/inform-about-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-002
+          uuid: ALA-1-0-0-op-s-bs-002
         - service-name: /v1/inform-about-release-history
-          uuid: xx-1-0-0-op-s-bs-003
+          uuid: ALA-1-0-0-op-s-bs-003
         - service-name: /v1/inform-about-release-history-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-004
+          uuid: ALA-1-0-0-op-s-bs-004
+        - service-name: /docs
+          uuid: ALA-1-0-0-op-s-bs-005
       individual:
-        - service-name: /v1/start-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-is-000
-        ## TODO: Add Services specific to this application
+        - service-name: /v1/provide-acceptance-data
+          uuid: ALA-1-0-0-op-s-is-000
   http-server:
-    own-application-name: ## TODO: Add OwnApplicationName
-    own-release-number: ## TODO: Add OwnReleaseNumber (e.g. 1.0.0)
-    uuid: xx-1-0-0-http-s-bm-000
-  tpc-server:
-    own-ip-address: ## TODO: Add OwnIpAddress
-    own-tcp-port: ## TODO: Add OwnTcpPort
-    uuid: xx-1-0-0-tcp-s-bm-000
-
+    own-application-name: AutomatedLinkAcceptance
+    own-release-number: 1.0.0
+    uuid: ALA-1-0-0-http-s-000
+  tcp-server:
+    - description: Without TLS layer
+      local-protocol: http
+      own-ip-address: 1.1.4.9
+      own-tcp-port: 4009
+      uuid: ALA-1-0-0-tcp-s-000
+    - description: With TLS layer
```

Comment:
https has it's another ip address and port than tcp. i.e. 1.2.4.9 and 4209. See [table](https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/TestingApplications/Infrastructure/SdnLaboratory/FakeAddresses/IpAddresses.md)

Reply:
The correct ip and tcp addresses for https were introduced